web-client/src/presenter/sequences/gotoTrialSessionsSequence.ts

```diff
@@ -31,5 +33,9 @@ export const gotoTrialSessionsSequence =
         setAllAndCurrentJudgesAction,
       ],
     ]),
+    [
+      getBulkSpecialTrialSessionCopyNotesAction,
+      setBulkSpecialTrialSessionCopyNotesAction,
+    ],
```

Comment:
This can run in the same parallel branch as `[getTrialSessionsAction, setTrialSessionsPageAction],`

Reply:
@TomElliottFlexion The trial session page hangs and never resolves when putting this in the parallel block

Reply:
@TomElliottFlexion more context here, we are depending on trialSessions to be loaded by `getTrialSessionAction` and loading that onto `props`. If I move the `getBulkTrialSessionCopyNotesAction` into the parallel block, it isn't guaranteed to be there.